sources/packages/web/src/components/common/students/StudentFileUploads.vue

```diff
@@ -23,63 +23,59 @@
       </body-header>
     </template>
     <content-group>
-      <toggle-content :toggled="!studentFileUploads.length">
-        <DataTable
-          :value="studentFileUploads"
-          :paginator="true"
-          :totalRecords="studentFileUploads?.length"
-          :rows="DEFAULT_PAGE_LIMIT"
-          :rowsPerPageOptions="PAGINATION_LIST"
+      <toggle-content
+        :toggled="!studentFileUploads.length"
+        message="No file uploads found."
+      >
+        <v-data-table
+          :headers="studentFileUploadHeaders"
+          :items="studentFileUploads"
+          :items-per-page="DEFAULT_PAGE_LIMIT"
+          :items-per-page-options="ITEMS_PER_PAGE"
+          :mobile="isMobile"
         >
-          <Column
-            field="groupName"
-            header="Document Purpose"
-            :sortable="true"
-          ></Column>
-          <Column
-            v-if="canViewUploadedBy"
-            field="uploadedBy"
-            header="Uploaded by"
-          >
-          </Column>
-          <Column field="metadata" header="Application #">
-            <template #body="slotProps">{{
-              emptyStringFiller(slotProps.data.metadata?.applicationNumber)
-            }}</template></Column
-          >
-          <Column field="createdAt" header="Date Submitted"
-            ><template #body="slotProps">{{
-              getISODateHourMinuteString(slotProps.data.createdAt)
-            }}</template></Column
-          >
-          <Column header="File">
-            <template #body="slotProps">
+          <template #loading>
+            <v-skeleton-loader type="table-row@5"></v-skeleton-loader>
+          </template>
+          <template #[`item.groupName`]="{ item }">
+            {{ item.groupName }}
+          </template>
+          <template v-if="canViewUploadedBy" #[`item.uploadedBy`]="{ item }">
```

Comment:
The v-if is technically required with conditional headers but I left in for clarity.